app/components/action-button.js

```diff
@@ -1,21 +1,33 @@
 //https://github.com/EnCiv/enciv-home/issues/8
 //https://github.com/EnCiv/enciv-home/issues/56
+//https://github.com/EnCiv/enciv-home/issues/55
+
 import React, { useState } from 'react'
 import { createUseStyles } from 'react-jss'
 import cx from 'classnames'
 import { Components } from 'civil-pursuit'
 import BrevoJoin from './brevo-join'
+import * as icons from '../svgr'
+
+function Iconify(props) {
+  const { iconName, ...otherProps } = props
+  const Icon = icons[iconName]
+  return <Icon {...otherProps} />
+}
 const { Button } = Components
 const ActionButton = props => {
   const {
     className = '', // may or may not be passed. Should be applied to the outer most tag, after local classNames
     action,
+    actionText = '',
```

Comment:
If it seems like new props need to be added that aren't in the issue definition, it needs to be discussed.  These props are how other components use this one, and they all have to coordinate.

I see the challenge, that the text being shown is in the children prop.  So this component is going to have to add the icon in front of of after the children, depending on the side.

you could pull children out of props and then render `<Button>{children}</Button>`  

I also wonder if you can do `children.push(<Icon>)` or children.shift(<Icon>) to put the icon on the left or the right.  But children might not take push or shift because it's a special react object - not really an array. 

Or you could try this
`const LeftIcon=side==='left' && iconName ? <Iconify ... /> | null` // similar for right icon
and then do 
`<Button><LeftIcon/>{children}<RightIcon/></Button>`